stroyprombeton/tests/tests_selenium.py

```diff
@@ -589,16 +589,17 @@ def test_search_have_results(self):
         self.assertTrue(self.browser.find_element_by_link_text('Category root #0'))
         self.assertTrue(self.browser.find_element_by_link_text('Category #0 of #1'))
 
+    # @todo #347:15m Move test_inactive_product_not_in_search_autocomplete to unit test.
```

Comment:
`to unit test` -> `Move test... to non-selenium format.`
Current format is unit test too